android/app/src/main/res/layout/music_player_widget.xml

```diff
@@ -0,0 +1,69 @@
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    style="@style/Widget.Android.AppWidget.Container"
+    android:theme="@style/Theme.Android.AppWidgetContainer"
+    android:id="@android:id/background"
+    android:background="@drawable/music_player_widget_background"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <ImageView
+        android:id="@+id/music_player_widget_no_song_art"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:importantForAccessibility="no"
+        android:scaleType="fitCenter"
+        android:src="@drawable/launch_foreground" />
+
+    <ImageView
+        android:id="@+id/music_player_widget_song_art"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:layout_centerInParent="true"
+        android:importantForAccessibility="no"
+        android:scaleType="centerCrop"
+        android:visibility="gone" />
+
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="@dimen/musicPlayerWidgetButtonSize"
+        android:layout_alignParentStart="true"
+        android:layout_alignParentEnd="true"
```

Comment:
```suggestion
        android:layout_width="wrap_content"
        android:layout_height="@dimen/musicPlayerWidgetButtonSize"
        android:layout_centerHorizontal="true"
```
We could also not stretch the button bar over the entire width, which would look like this:

![Widgets with centered button bar](https://github.com/user-attachments/assets/8e712fd6-6e5b-495a-a0a8-a1d25914111d)
![More widgets with centered button bar](https://github.com/user-attachments/assets/a453ea20-74d8-42cc-aa88-9562860bcef2)

Would you prefer that?

Reply:
I think the stretched one looks nicer